extensions/easyvariable/src/utils/translators/raycast.ts

```diff
@@ -0,0 +1,26 @@
+import { AI } from "@raycast/api";
+import { preferences } from "../preferences";
+
+export const raycastTranslate = async (text: string): Promise<string> => {
+  if (!text.trim()) return "";
+
+  if (!preferences.enableRaycastTranslate) {
+    return "";
+  }
+
+  try {
+    // Use Raycast AI API for translation
+    const response = await AI.ask(
+      `Translate the following text to English. Only return the translated text without explanation or punctuation. Use common abbreviations and technical terms where appropriate:\n\n${text}`,
+    );
```

Comment:
Youc an use `enviroment.canAccess(AI)` for checking the user's permission to Raycast AI.

See https://developers.raycast.com/api-reference/environment#environment.canaccess.